core/auth/fake/interfaces/controller.go

```diff
@@ -0,0 +1,186 @@
+package interfaces
```

Comment:
Remove the `interfaces` folder, move all to `core/auth/fake`.

Reply:
done